TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robots/Callisto.java

```diff
@@ -217,6 +170,90 @@ private void initTeleOp(Gamepad gamepad1, Gamepad gamepad2) {
                     intakeMotorSubsystem.stop();
                 }));
 */
+
+
+
+        //region Utility Button B
+        // linear slide extends to high position
+        utilityGamepad.getGamepadButton(GamepadKeys.Button.B)
+                .whenPressed(
+                        new SequentialCommandGroup(
+                                new RunLinearSlideAndCenterPixelBoxCommand( extakeSubsystem,linearSlideSubsystem, Configuration.LINEAR_SLIDE_POS_HI),
+                                new MovePixelBoxArmToPositionCommand(extakeSubsystem, PixelBoxArmPosition.Extake)
+                        )
+                );
+        //endregion
+
+        //region Utility Button Y
+        // linear slide extends to med position
+        utilityGamepad.getGamepadButton(GamepadKeys.Button.Y)
+                .whenPressed(
+                        new SequentialCommandGroup(
+                                new RunLinearSlideAndCenterPixelBoxCommand(extakeSubsystem,linearSlideSubsystem, Configuration.LINEAR_SLIDE_POS_MED),
+                                new MovePixelBoxArmToPositionCommand(extakeSubsystem, PixelBoxArmPosition.Extake)
+                        )
+                );
+        //endregion
+
+        //region Utility Button X
+        //linear slide extends to low position
+        utilityGamepad.getGamepadButton(GamepadKeys.Button.X)
+                .whenPressed(
+                        new SequentialCommandGroup(
+                                new RunLinearSlideAndCenterPixelBoxCommand(extakeSubsystem,linearSlideSubsystem, Configuration.LINEAR_SLIDE_POS_LO),
+                                new MovePixelBoxArmToPositionCommand(extakeSubsystem, PixelBoxArmPosition.Extake)
+                        )
+                );
+        //endregion
+
+        //region Utility D-Pad Left
```

Comment:
I think we should break up the move of the pixel box and the direction of it.  So d-pad moves the pixel box and then the left-stick can find tune it.  we can then make the right bumper drop the pixel.  I would also consider making the linear slide move up and down with this left joystick so you can fine tune the location when dropping